lib/kubernetes-deploy/kubeclient_builder.rb

```diff
@@ -59,5 +67,10 @@ def _build_kubeclient(api_version:, context:, endpoint_path: nil)
       client.discover
       client
     end
+
+    def config_files
+      # Split the list by colon for Linux and Mac, and semicolon for Windows.
+      ENV.fetch("KUBECONFIG").tr(":;", " ").split
```

Comment:
Nitpick: 
```ruby
ENV.fetch("KUBECONFIG").split(/[:;]/)
```

Reply:
Revised as suggested and added code to trim spaces and remove empty items from the array of file names.